certified-connectors/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API \u91d1\u9470",
+          "description": "\u6b64 API \u7684 API \u91d1\u9470",
+          "tooltip": "\u63d0\u4f9b\u60a8\u7684 API \u91d1\u9470",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#DDDDDD",

```

Comment:
@ccyy915 Please make sure that you are following [instructions](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#icon) for icon while making submission in the certification portal. And the `iconBrandColor` should be background color of your connector's icon.